### PR TITLE
Fix docs and regenerate scaffolding.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -21,75 +21,19 @@ library's README.
 
 ## Samples
 
-### Logging
+### Quickstart
 
-View the [source code][logs_0_code].
+View the [source code][quickstart_0_code].
```
Comment (gaofanmichael) [inline samples/README.md]: The preview of this part under samples is a little weird. Could you please verify the layout in a correct format?

File: samples/README.md [hunk 2]
```diff
@@ -21,75 +21,19 @@ library's README.
...
-  bunyan-setup                        Setup up the Bunyan logger with explicit credentianls.
-  winston                             Writes some logs entries to Stackdriver Logging via Winston.
-  winston-setup                       Setup up the Winston logger with explicit credentianls.
-  delete <logName>                    Deletes the specified Log.
+### Explict Auth Setup
 
-Options:
-  --version  Show version number                                                                               [boolean]
-  --help     Show help                                                                                         [boolean]
+View the [source code][setup_explicit_1_code].
```
Comment (gaofanmichael) [inline samples/README.md]: Same here.

File: samples/README.md (hunk 1 above)
Comment (jmdobry) [reply samples/README.md]: Fixed.

File: samples/README.md (hunk 2 above)
Comment (jmdobry) [reply samples/README.md]: Fixed.

**Pull request merged**